### PR TITLE
Do not show notification for non-confirmed bookings

### LINKED ISSUE #375: Notification your flight departs in X hours also shown for refunded booking
Info here https://skypicker.slack.com/archives/CAZ5BCEE8/p1528387969000793

### PR DESCRIPTION
**Pull request by jaroslav-kubicek**
closes #375 <br/><br/><br/><url>LiveURL: https://smartfaq-375-notification-update.surge.sh</url>

### CODE REVIEW

File: src/SingleBookingPage/__tests__/BookingDetail.test.js [hunk 1]
```diff
@@ -58,6 +59,13 @@ describe('BookingDetail >', () => {
       expect(wrapper.find(Notification).props().isUrgent).toBeTruthy();
     });
 
+    it('is not displayed when status is anything but CONFIRMED', () => {
+      // $FlowExpectedError: We don't need all props for tests
+      const refundedBooking = { ...booking48HoursAgo, status: 'REFUNDED' };
```
Comment (kantord) [inline src/SingleBookingPage/__tests__/BookingDetail.test.js]: hmm maybe we could test for several different values of status here? I'm thinking of something like this:
```
['REFUNDED', 'CANCELLED', ...].forEach(status => it(`is not displayed when status is anything but ${status}`, () => {
  ... rest of the test ...
})
```

```

**Comment by kantord** [comment]
Looks good!

File: src/SingleBookingPage/__tests__/BookingDetail.test.js (hunk 1 above)
Comment (jaroslav-kubicek) [reply src/SingleBookingPage/__tests__/BookingDetail.test.js]: 🤔  we can do it, but does it worth? it's quite much for one condition

Comment (kantord) [reply src/SingleBookingPage/__tests__/BookingDetail.test.js]: Yeah, not sure :smiley:...

**Pull request merged**